### DISCUSSION TITLE
Icons not showing in the Rapidpro new UI

### DISCUSSION THREAD

**Discussion by mnjenga** [nyaruka/rapidpro#4504] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/9023743/233690369-8dc04a91-605e-4f1a-a895-97f99e186a4a.png)
![image](https://user-images.githubusercontent.com/9023743/233690493-b5b4d547-bc10-4ca0-937e-2d53c09d6028.png)

I am experimenting with the new Rapidpro UI and I am struggling to get the temba-components Icons to display(1st image shows how they should appear and the second image shows how my installation looks like). Anyone who can guide on what I am missing

**Comment by ericnewcomer** [comment, marked as answer]
I've seen this before a couple times. For me it was one of the following:

  1) Static files are not available for the `temba-components` installed in `node_modules` for some reason
  2) Local web server is truncating the SVG file due to its size when serving it
  
I'm actually working on a PR right now that prunes the size of the icon set down to only what is used which could help if your issue is the latter. In either case, you should be able to address this in your install with appropriate settings though.

**Reply by mnjenga** [reply]
Thanks for pointing me in the right direction, in my case, the file was being served with the wrong mime type, and updating "image/svg+xml svg svgz;" on NGINX resolved the issue
